nomad/leader.go

```diff
@@ -532,8 +532,10 @@ func (s *Server) reapFailedEvaluations(stopCh chan struct{}) {
 			// due to the fairly large backoff.
 			followupEvalWait := s.config.EvalFailedFollowupBaselineDelay +
 				time.Duration(rand.Int63n(int64(s.config.EvalFailedFollowupDelayRange)))
+
 			followupEval := eval.CreateFailedFollowUpEval(followupEvalWait)
 			updateEval.NextEval = followupEval.ID
+			updateEval.ModifyTime = time.Now().UTC().UnixNano()
```

Comment:
evals stored in the raft log by earlier versions will have `CreateTime` set to 0, do we need to set it if that's the case? Assuming this change primarily supports logging, that may be fine.

Reply:
@langmartin good catch, I'd say its fine to leave CreateTime as zero for evals from older versions of Nomad because we can't set it to a correct value. And ys, this change is primarily for logging/CLI debugging.